spec/features/editing_images/upload_lead_image_spec.rb

```diff
@@ -13,11 +13,11 @@
 
   def given_there_is_a_document
     document_type = build(:document_type, lead_image: true)
-    create(:document, document_type_id: document_type.id)
+    @document = create(:document, document_type_id: document_type.id)
```

Comment:
Perhaps split the changes to this file out too as it's not related to the commit message?